### DISCUSSION TITLE
[suggestion] allow users to choose custom nickname colors

### DISCUSSION THREAD

**Discussion by fevers** [element-hq/element-meta#653] [Ideas]
Currently, nickname colors are assigned based on the person's username, and can't be changed except by starting over with a new account. It would be nice if one could instead enter the hex for one's preferred color somewhere in the settings page, and have the old color replaced with the preferred one.

**Comment by q-TheDog** [comment]
+1 for this as well as letting room admins set room-level colors based on PL (overriding users' custom colors in that room should they have one set)
